torch/nn/functional.py

```diff
@@ -556,6 +556,22 @@ def batch_norm(input, running_mean, running_var, weight=None, bias=None,
     return f(input, weight, bias)
 
 
+def layer_norm(input, weight=None, bias=None, eps=1e-5):
```

Comment:
we should also move the dim checks here, as this can be called outside of the module.